packages/web/src/components/gcds-link/test/gcds-link.spec.ts

```diff
@@ -45,10 +45,10 @@ describe('gcds-link', () => {
   it('[french] renders external link with label + icon', async () => {
     const { root } = await newSpecPage({
       components: [GcdsLink],
-      html: `<gcds-link href="https://google.com" target="_blank" lang="fr">External Link</gcds-link>`,
+      html: `<gcds-link href="https://google.com" external lang="fr">External Link</gcds-link>`,
     });
     expect(root).toEqualHtml(`
-      <gcds-link href="https://google.com" target="_blank" lang="fr">
+      <gcds-link href="https://google.com" external lang="fr">
```

Comment:
We already have it added to the `index.html` below, but can we also add it to the test? Let's add one text for:
```javascript
// This link opens in a new tab, but is not external.
<gcds-link target="_blank"> 
```